nzedb/MiscSorter.php

```diff
@@ -265,49 +304,67 @@ private function doOS($nfo = '', $id = 0)
 				$set[1] = $tmp[1];
 			}
 			if (strlen($set[1]) < 128 && !preg_match('/(another)? *(fine)? *release/i', $set[1])) {
-				$ok = $this->dodbupdate($id, $this->cleanname($set[1]), null, "app");
+				$ok = $this->dodbupdate($id, $this->cleanname($set[1]), null, 'app');
 			}
 		}
 
 		return $ok;
 	}
 
+	/**
+	 * @param string $pattern
+	 * @param string $nfo
```

Comment:
If these types are absolutes, we should type the parameters in the signature too.